cmd/cloud/organization.go

```diff
@@ -113,11 +109,7 @@ func newOrganizationExportAuditLogs(_ io.Writer) *cobra.Command {
 			return organizationExportAuditLogs(cmd)
 		},
 	}
-	cmd.PersistentFlags().StringVarP(&orgName, "organization-name", "n", "", "Name of the Organization to manage audit logs for.")
```

Comment:
all I changed here is make organization-name no longer required. Shouldn't break anything